recipes/ijava/meta.yaml

```diff
@@ -0,0 +1,89 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Ping

Reply:
Pong. Thank you for the reminder. Comments removed in https://github.com/conda-forge/staged-recipes/commit/60228f19bb03adc3849ffac8abba04768a5feea9